src/style.css

```diff
@@ -33,6 +33,15 @@ p {
 	line-height: 1.65;
 }
 
+button {
```

Comment:
so much styling on an HTML tag isn't good to have, as we might not want all `button` tags to like that. Just keeping `font-family` and `font-size` in button tag.